DesignSafe-CI/portal
# task/DES-2638 - Recon Portal Header Update

## Overview: ##
NSF and PIs have requested a change to the Recon Portal front end to provide users with clear instructions. We've updated the validators for the admin form, and the app should no longer accept impossible Lat and Long coordinates.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2638](https://tacc-main.atlassian.net/browse/DES-2638)

## Summary of Changes: ##
Added a header to Recon Portal that includes instructions on how to use the app and moves the logo out of the sidebar.

## Testing Steps: ##
1. Go to https://designsafe.dev/recon-portal/
2. Make sure the header looks like the photos below.
3. Make sure the portal works properly
4. Go to https://www.designsafe.dev/recon-portal/admin
5. Try and enter an impossible Lat & Long (eg: (-90, 90)).
6. The form should prevent submission.

## UI Photos:
![Screenshot 2024-05-01 at 9 33 15 AM](https://github.com/DesignSafe-CI/portal/assets/7462330/2a33a575-61b3-41c6-abde-cd8cb4932f84)

## Notes: ##
